src/adapters/CKEditorAdapter.ts

```diff
@@ -35,11 +41,21 @@ export class CKEditorAdapter extends AbstractRichtextEditorAdapter {
   }
 
   getEditorDocument(): Document {
-    return this.getEditor().document.$ as any;
+    // for version 4
+    if(this.editorVersion == 4){
+      return this.getEditor().document.$ as any;
+    }
+    // for version 5
+    return document.querySelector(this.editorIdHashed)!.ownerDocument;
   }
 
   getContent() {
-    return this.getEditor().getData();
+    // for version 4
+    if(this.editorVersion == 4){
+      return this.getEditor().getData();
+    }
+    // for version 5
+    return document.querySelector(this.editorIdHashed)!.innerHTML;
```

Comment:
Hi,
The purpose of the adapter is to use editor API to fetch content. If we simply use HTML dom operation to get content there is no need to adapter.

If you look a v4 implementation it gets CK4 editor instance and uses getData() API of the editor.

Getting content via DOM can already be done using contenEditableAdapter. https://github.com/acrolinx/sidebar-sdk-js/blob/main/src/adapters/ContentEditableAdapter.ts#L35

I think we should use CK5 API to get content from editor and not DOM innerHTML